_conferences/devoxx-be-2018.md

```diff
@@ -0,0 +1,12 @@
+---
+name: "Devoxx"
+website: https://devoxx.be/
+location: Antwerp, Belgium
+
+date_start: 2018-11-12
+date_end:   2018-11-16
+
+cfp_start: 2018-05-31
+cfp_end:   2018-07-06
+cfp_site: https://cfp.devoxx.be/
```

Comment:
That link does not seem to work for me. It's loading endlessly.

Reply:
#400